astronomer/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -0,0 +1,73 @@
+from aiobotocore.client import AioBaseClient
+from aiobotocore.session import get_session
+from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook, _parse_s3_config
+from asgiref.sync import sync_to_async
+
+
+class AwsBaseHookAsync(AwsBaseHook):
+    """
+    Interacts with AWS using aiobotocore asynchronously.
+
+    .. note::
+        AwsBaseHookAsync uses aiobotocore to create asynchronous S3 hooks. Hence, AwsBaseHookAsync
+        only supports the authentication mechanism that aiobotocore supports. The ability to assume
+        roles provided in the Airflow connection extra args via aiobotocore is not supported by the
+        library yet.
+
+    :param aws_conn_id: The Airflow connection used for AWS credentials.
+        If this is None or empty then the default boto3 behaviour is used. If
+        running Airflow in a distributed manner and aws_conn_id is None or
+        empty, then default boto3 configuration would be used (and must be
+        maintained on each worker node).
+    :param verify: Whether or not to verify SSL certificates.
+        https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html
```

Comment:
You can use the seealso Sphinx directive for this link - https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-seealso

Reply:
Disable flake8 `RS303` check since it was giving false positive and moved the documentation ref at end of param since multiple param pointing to the same link

Reply:
Docs screenshot

<img width="737" alt="Screenshot 2022-06-24 at 11 21 03 PM" src="https://user-images.githubusercontent.com/98807258/175615646-a6d534c3-a56d-4231-ac9e-ef7f15fed180.png">